src/ruby/spec/spec_helper.rb

```diff
@@ -0,0 +1,97 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
+# specs live under a `spec` directory, which RSpec adds to the `$LOAD_PATH`.
+# The generated `.rspec` file contains `--require spec_helper` which will cause
+# this file to always be loaded, without a need to explicitly require it in any
+# files.
+#
+# Given that it is always loaded, you are encouraged to keep this file as
+# light-weight as possible. Requiring heavyweight dependencies from this file
+# will add to the boot time of your test suite on EVERY test run, even for an
+# individual file that may not need all of that loaded. Instead, consider making
+# a separate helper file that requires the additional dependencies and performs
+# the additional setup, and require it from the spec files that actually need
+# it.
+
+#Loading the classes to be tested
+require_relative '../lib/symengine'
```

Comment:
Ah, now I remember :relieved: . Thanks for pointing it out :smile:  Personally, I wouldn't like the user to install the gem if he/she wants to run the tests. So, the second one is better.
What you suggested is very similar to this: https://github.com/v0dro/daru/blob/master/daru.gemspec#L2. But this is in gemspec. I came through this while reviewing @v0dro's code. 

Both should work. What is the standard/preferred way to do this?